cmds/mount/mount.go

```diff
@@ -25,14 +27,38 @@ var (
 	opt    = flag.String("o", "", "Specify mount options")
 )
 
+func informIfUnknownFS(originFS string) {
+	file, err := os.Open("/proc/filesystems")
+	if err != nil {
+		// just don't make things even worse...
+		return
+	}
+	defer file.Close()
```

Comment:
I would generally recommend checking the error and dealing with it if there is an error (even if it's just a log).

Reply:
in this one case, a readonly file, I don't see a need. Point taken, but this is a very common idiom in Go. There appear to be 900 instances of it in the Go source tree.